Extensions/FixMyPosts.js

```diff
@@ -0,0 +1,64 @@
+//* TITLE FixMyPosts **//
+//* VERSION 999.999.999 **//
+//* DESCRIPTION Fixes the reblog format of posts to reflect the blockquote-style **//
+//* DEVELOPER THETIMEISNEVERIGHT **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.FixMyPosts = new Object({
+
+    running: false,
+
+    preferences: {
+        "sep-0": {
+            text: "Please use Reblog Display Options instead of FixMyPosts",
+            type: "separator"
+        },
+    },
+
+    run: function() {
+        this.running = true;
+
+        XKit.window.show("Use Reblog Display Options instead of FixMyPosts",
+            "<p>FixMyPosts is rather buggy and a third party extension, "+
+            "not supported by the New XKit Team. Reblog Display Options has"+
+			"the same functionality and is an official extension. You"+
+			"just need to set RDO's style to nested in the RDO preferences.</p>"+
+            "<br>Would you like to install Reblog Display Options?",
+            "info",
+            '<div id="xkit-install-br" class="xkit-button default">Install Reblog Display Options</div>'+
+            '<div id="xkit-close-message" class="xkit-button">Don\'t install</div>');
+        $("#xkit-install-br").click(function(){
+            XKit.install("better_reblogs", function(mdata) {
+                if (mdata.errors) {
+                    if (mdata.storage_error === true) {
+                        show_error_installation("[Code: 631] Can't store data on browser");
+                        return;
+                    }
+                    if (mdata.server_down === true) {
+                        show_error_installation("[Code: 101] Can't reach XKit servers");
```

Comment:
should both specifically reference "error installing RDO" (and link to new-xkit-support? not sure if show_error_installation does this or not)